change/@ni-nimble-angular-204fcb18-d5bb-4ad9-bb52-53bda9837f75.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I think we should call this `major`. It will break the build of any client on Angular 15 unless they update to 16 so it counts as an API-breaking change.